### PR TITLE
REPORT-343020: Include helm file changes

### PR DESCRIPTION
**Pull request by Subasri-Viswanathan**
Included helm chart changes in v5.4.20
Updated the max replicas to 1 in the report service and jobs due to the export issue on the designer.

### CODE REVIEW

File: helm/README.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+NAME            CHART VERSION   APP VERSION     DESCRIPTION
+boldreports/boldreports   5.4.20           5.4.20         Make bolder business decisions with complete reporting solutions...
+```
+
+_See [helm repo](https://helm.sh/docs/helm/helm_repo/) for command documentation._
+
+## Install Chart
+
+For Helm chart, you'll need to craft a `values.yaml`.
+
```
Comment (ranganathan-arumugam) [inline helm/README.md:37]: add a gke values.yaml file section too.

File: README.md [hunk 2]
```diff
@@ -44,3 +44,9 @@ These pre-built images are provided for convenience and include all optional and
 If you want to install Bold Reports from scratch and precisely control which optional libraries are installed, please download the stand-alone product from boldreports.com. If you have any questions, please contact the Bold Reports team (https://www.boldreports.com/support).
 
 It is the image user's responsibility to ensure that any use of this image complies with any relevant licenses for all software contained within.
+
+## FAQ
+
+[How to auto deploy Bold Reports in Kubernetes cluster?](https://github.com/boldreports/bold-reports-kubernetes/blob/master/docs/bold-reports-auto-deployment.md)
```
Comment (ranganathan-arumugam) [inline README.md:50]: Refer the files from their same branch

File: helm/README.md (hunk 1 above)
Comment (Subasri-Viswanathan) [reply helm/README.md:37]: Included gke related changes

File: README.md (hunk 2 above)
Comment (Subasri-Viswanathan) [reply README.md]: Changed

**Comment by ranganathan-arumugam** [comment]
Remove the auto deployment md file as of now. we will include once IDP fix the issue in auto deployment.

File: docs/FAQ/how-to-deploy-bold-reports-in-eks-using-application-load-balancer.md [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+8. Open the **deploy_eks_alb.yaml** file that was downloaded in Step 6. Replace the file system ID that was created in Step 2 with the `<efs_file_system_id>` placeholder in the file.
+
+   ![PV Claim](./images/alb_efs_file_system.png)
+
+9. Enter the necessary variable information in the **deploy_eks_alb.yaml** file to complete the auto-deployment, as detailed below.
+
+    * Enter the Bold Reports license key, user, and database server details.
+	The following environment variables are optional. If not provided, a manual Application start-up configuration is needed.
+        
+        ![Licence-and-User-Details](../images/deploy/eks/licence-and-user-details.png)
```
Comment (ranganathan-arumugam) [inline docs/FAQ/how-to-deploy-bold-reports-in-eks-using-application-load-balancer.md:47]: This is image was broken give proper path.

File: helm/README.md [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+* For `EKS` please download the values.yaml file [here](https://raw.githubusercontent.com/boldreports/bold-reports-kubernetes/v5.4.20/helm/custom-values/eks-values.yaml).
```
[the remark this answers is not in the document]
Comment (Subasri-Viswanathan) [reply helm/README.md:38]: referred the master branch

File: docs/FAQ/how-to-deploy-bold-reports-in-eks-using-application-load-balancer.md (hunk 3 above)
Comment (Subasri-Viswanathan) [reply docs/FAQ/how-to-deploy-bold-reports-in-eks-using-application-load-balancer.md]: removed the auto deployment related md

**Review by Sankaralingam1667 — APPROVED** [review]

**Pull request merged**